deployment/ansible/deploy.yml

```diff
@@ -61,7 +61,7 @@
           port: 3114
       chain:
         persist: true
-        db_path: "./db43"
+        db_path: "./db44"
```

Comment:
This will conflict with https://github.com/aeternity/epoch/pull/1018/files#diff-2ad1947fe9b869ffd23a6a86872e65e0R64

Reply:
Removed.